### ISSUE TITLE
Testing procedure

### ISSUE THREAD

**Issue by peterwebster** [ukwa/w3act#197]
Once @kinmanli is happy that #195 is complete. need to test UI in following order:

(i) check that all default views display correctly and show the expected data;

(ii) test all search functions, with each of the applicable filters applied in turn, for correctness of data returned;

(iii) check Open UKWA licence process from start to finish;

(iv) create and populate some target records, using multiple user roles;

(v) test QA workflows

(vi) add, amend records in each taxonomy (collections, subjects etc)

After this, we can moved to double-entry of data to W3ACT and Andy's ACT, as per #182 

cc @anjackson

**Comment by peterwebster** [comment]
(i) noticing that the list of targets default to showing none (rather than all) at http://www.webarchive.org.uk/actdev/targets/list?s=title  Probably a slight improvement, so let's leave it so.

**Comment by peterwebster** [comment]
(ii) target list view and the application of filters tested, new tickets raised.

**Comment by peterwebster** [comment]
(ii) tested the Target Creation tab in Reports, two new tickets #221 and #222 
Need to leave the QA and UKWA licence tabs until have created some test data.

**Comment by peterwebster** [comment]
Tested the Subject, Collection and Tag taxonomy views, and successfully added new values to each. 

Question: should Archivist be able to add/edit flag values in Taxonomies > Flags  ? I think so, but not urgent.

**Comment by peterwebster** [comment]
Tested the Users&Agencies > Users and Organisation creating new tickets #225 and #226 

Need to test Organisations and Roles when logged in as SysAdmin

**Comment by peterwebster** [comment]
Should be able to return to this on Friday @anjackson @kinmanli 

Aiming to have completed this initial full test of the whole UI by Monday or possibly Tuesday.

**Comment by peterwebster** [comment]
Part (iii) postponed, pending #229

**Comment by peterwebster** [comment]
Testing UKWA permissions, tickets #234, #239

Able to queue a permission, send out an email, reject an email from queue.

**Comment by peterwebster** [comment]
As we've now gone into production, closing this ticket, as anything missed here will now need a new ticket.
